### PR TITLE
reword "data set" -> "dataset"

### PR DESCRIPTION
**Pull request by thoniTUB**
Before:  
image

After:
image

### CODE REVIEW

**Comment by thoniTUB** [comment]
I would prefer `Dataset` which is okay according to https://en.wiktionary.org/wiki/dataset, but other sources say it's less academic

File: frontend/src/localization/en.json [hunk 1]
```diff
@@ -82,10 +82,10 @@
     "aConceptList": "a concept list"
   },
   "datasetSelector": {
-    "label": "Data sets",
```
Comment (awildturtok) [inline frontend/src/localization/en.json:85]: Dataset ist ein echtes Wort, warum wird das überhaupt gesplittet?

Comment (Kadrian) [reply frontend/src/localization/en.json:85]: I agree, let's just use dataset as one word

Comment (thoniTUB) [reply frontend/src/localization/en.json:85]: 👍

**Pull request merged**